pages/pipelines/skipping.md.erb

```diff
@@ -0,0 +1,77 @@
+# Skipping builds
+
+Build skipping allows you to avoid unnecessary rebuilds, conserving resources and freeing up agents.
+
+{:toc}
+
+## Skip queued intermediate builds
+
+Sometimes you may push several commits in quick succession, leading to Buildkite building each commit in turn. You can configure your pipeline to always skip these intermediate builds, and only build the latest commit.
+
+To skip pending builds on the same branch:
+
+1. Navigate to your *Pipeline Settings*.
+2. Select *Builds*.
+3. Select *Skip Intermediate Builds*.
+4. **Optional**: limit which branches build skipping applies to by adding branch names in the text box below *Skip Intermediate Builds*. For example, "branch-one" means Buildkite only skips intermediate builds on branch-one. You can also use not-equals: "!main" skips intermediate builds on all branches except main.
+
+You can also configure these options using the [REST API](/docs/apis/rest-api/pipelines#create-a-yaml-pipeline).
+
+## Cancel running intermediate builds
+
+Sometimes you may push several commits in quick succession, leading to Buildkite building each commit in turn. You can configure your pipeline to always cancel any running builds, and only build the latest commit.
+
+To cancel running builds on the same branch:
+
+1. Navigate to your *Pipeline Settings*.
+2. Select *Builds*.
+3. Select *Cancel Intermediate Builds*.
+4. **Optional**: limit which branches build skipping applies to by adding branch names in the text box below *Cancel Intermediate Builds*. For example, "branch-one" means Buildkite only skips intermediate builds on branch-one. You can also use not-equals: "!main" skips intermediate builds on all branches except main.
+
+You can also configure these options using the [REST API](/docs/apis/rest-api/pipelines#create-a-yaml-pipeline).
+
+## Manually cancel jobs and skip to next step
+
+If your pipeline has multiple command steps, you can manually cancel a step, and Buildkite will proceed to the next step. This does not cause the build to fail.
```

Comment:
@plaindocs We think this is untrue, unless:
- `allow_failure` and/or `soft_fail` are in effect, or
- the command step exits successfully (`exit(0)`) despite receiving a cancellation signal.